app/services/canvas_service.py

```diff
@@ -7,12 +7,18 @@
 from pathlib import Path
 from enum import Enum
 from urllib.parse import urlparse
-from pydantic import BaseModel
+from pydantic import BaseModel, PositiveInt
 from datetime import datetime
 from fastapi import HTTPException
 from sqlalchemy.orm import Session
 from app.core.config import settings
+from app.core.exceptions.assignment import AssignmentNotPublishedException
+from app.enums.assignment_status import AssignmentStatus
+from app.enums.canvas.canvas_workflow_state_filter import CanvasWorkflowStateFilter
 from app.models import UserModel, OnyenPIDModel
+from app.models.assignment import AssignmentModel
+from app.models.course import CourseModel
+from app.models.user.student import StudentModel
```

Comment:
above imports aren't used anywhere in this file I think

Reply:
Fixed